front-packages/akeneo-design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {forwardRef, Ref, HTMLAttributes, useState} from 'react';
+import styled from 'styled-components';
+import {Override} from '../../shared';
+import {HelpPlainIcon} from '../../icons';
+import {AkeneoThemedProps, getColor, getFontSize} from '../../theme';
+import {useTheme} from '../../hooks';
+
+const TooltipMargin = '5px';
+
+const TooltipContainer = styled.div`
+  position: relative;
+  display: inline-block;
+`;
+
+const TooltipTitle = styled.div`
+  font-weight: 600;
+  margin-bottom: 5px;
+`;
+
+const TooltipContent = styled.div<{direction: string} & AkeneoThemedProps>`
+  position: absolute;
+  border-radius: 4px;
+  left: 50%;
+  padding: 10px;
+  color: ${getColor('blue', 120)};
+  background: ${getColor('blue', 10)};
+  border: 1px solid ${getColor('blue', 40)};
+  font-size: ${getFontSize('default')};
+  line-height: 1;
+  z-index: 100;
+  white-space: nowrap;
+  box-shadow: 0px 0px 16px rgba(89, 146, 199, 0.25);
+
+  ${({direction}) => {
+    switch (direction) {
+      case 'top':
+        return `
+                bottom: calc(100% + ${TooltipMargin});
+                transform: translateX(-50%);
+              `;
+      case 'bottom':
+        return `
+                top: calc(100% + ${TooltipMargin});
+                transform: translateX(-50%);
+              `;
+      case 'left':
+        return `
+                left: auto;
+                top: 50%;
+                right: calc(100% + ${TooltipMargin});
+                transform: translateY(-50%);
+              `;
+      case 'right':
+        return `
+                top: 50%;
+                left: calc(100% + ${TooltipMargin});
+                transform: translateY(-50%);
+              `;
+      default:
+        return `
+                bottom: calc(100% + ${TooltipMargin});
+                transform: translateX(-50%);
+              `;
+    }
+  }}
+`;
+
+type TooltipProps = Override<
+  HTMLAttributes<HTMLDivElement>,
+  {
+    direction?: 'top' | 'right' | 'bottom' | 'left';
```

Comment:
Could you add description for each prop plz?
See other component, description should be added even if it could be pretty obvious

For example:

```suggestion
    /**
     * Define the direction in which the tooltip will be rendered
     */
    direction?: 'top' | 'right' | 'bottom' | 'left';
```